CONTRIBUTING.rst

```diff
@@ -0,0 +1,55 @@
+Contributing
+============
+
+We welcome your contributions to Kagi and strive to make it as easy as possible
+to participate.
+
+Initial Set-up
```

Comment:
I would add a part:

# The quick way

    make install

# The long ride

....

Reply:
I agree that we should make it easy to get started. I intentionally left the "quick-start" section out as I considered what to do regarding #24. I'll post a follow-up comment there in order to get additional clarity regarding our goals.

Reply:
I posted some questions as a comment to the aforementioned issue regarding what we want our goals to be. For now, I'd like to merge this and iterate on it after we have some consensus on said goals. :smile: